modules/installation-configuration-parameters.adoc

```diff
@@ -677,6 +677,10 @@ ifdef::gcp[If you are installing on GCP into a shared virtual private cloud (VPC
 ====
 Not all CCO modes are supported for all cloud providers. For more information about CCO modes, see the _Cloud Credential Operator_ entry in the _Cluster Operators reference_ content.
 ====
+[NOTE]
+====
+If your AWS account has SCP (Service control policies) enabled, you must modify `install-config.yaml` file and add the `credentialsMode: Mint`, `credentialsMode: Passthrough` or `credentialsMode: Manual` entry.
+====
 |`Mint`, `Passthrough`, `Manual`, or an empty string (`""`).
```

Comment:
@xenolinux What does `""` mean here? 
It means no entry for credentialsMode or "credentialsMode: "?

Reply:
I guess it means no entry for the `credentialsMode`.

Reply:
`credentialsMode: ""` equals _NO `credentialsMode` entry in install-config_, it's the default mode.

Reply:
@yunjiang29 Did you ever run credentialsMode: "" for SCP enable case? We never tested it but Mingxia can have a try.

Reply:
Mingxia just had a try. Yes, credentialsMode: "" equals to no entry so the install failed when SCP is enabled.
@xenolinux We need to delete the part of ', or an empty string (`""`).'

Reply:
@jianping-shu Addressed- removed the part "or an empty string ("")".

Reply:
@xenolinux Pls. roll back this change per my comment. This value list should be the full list and not related to SCP issue.